datacube/api/grid_workflow.py

```diff
@@ -29,7 +29,7 @@
     from odc.geo.geobox import GeoBox
 
     from datacube.index import Index
-    from datacube.model import Product
+    from datacube.model import GridSpec, Product
```

Comment:
I think it might make more sense for `datacube.model.GridSpec` to be the aliased import, given it's deprecated